v2/cmd/asoctl/pkg/importresources/resource_importer.go

```diff
@@ -207,7 +242,6 @@ func (ri *ResourceImporter) importWorker(
 	completed chan<- ImportResourceResult,
 	failed chan<- ImportError,
 	progress importreporter.Interface,
-	done chan<- struct{},
```

Comment:
Need to update fn doc to remove `done` channel

Reply:
Done.